### DISCUSSION TITLE
comparison of ESM-1b and ESM-1v

### DISCUSSION THREAD

**Discussion by xf-omics** [facebookresearch/esm#129] [Q&A] [answered]
First of all, congratulations on your great work and thank you very much for sharing your models with the research community.

I have a question about the performance of ESM-1b and ESM-1v models. In the Meier et al. 2021 preprint, the predictive performance is better using UniRef90 compared with UniRef50. However, in your github "Comparison to related works" table, ESM-1b using UniRef 50 is better than ESM-1v using UniRef 90. Could you advise which model is better if we would like to use your models for downstream predictions. Thank you!

**Comment by joshim5** [comment, marked as answer]
Hi @fancy1228, we've found that performance depends on the downstream task. If you're interested in function prediction, use ESM-1v (trained on Uniref90). If you're interested in structure prediction, use ESM-1b (trained on Uniref50). For other applications, you may want to experiment with representations from both models.
